javalib/src/main/scala/java/util/concurrent/ThreadLocalRandom.scala

```diff
@@ -0,0 +1,81 @@
+/*
+ * closely translated from:
+ * https://github.com/scala/scala
+ * scala.concurrent.fokjoin.ThreadLocalRandom
+ */
+
+package java.util.concurrent
+
+import java.util.Random
+
+class ThreadLocalRandom extends Random {
+
+  override def setSeed(seed: Long): Unit =
+    if (initialized) throw new UnsupportedOperationException()
+    else super.setSeed(seed)
```

Comment:
This if is in statement position. It should be expanded with its side-effects on dedicated lines:
```scala
... = {
  if (initialized)
    throw new ...
  else
    super.setSeed(seed)
}
```
or without the else:
```scala
... = {
  if (initialized)
    throw new ...
  super.setSeed(seed)
}
```